src/pps/filter.go

```diff
@@ -0,0 +1,22 @@
+package pps
+
+import (
+	context "context"
```

Comment:
This happens in my environment as well. It seems like it must be a goimports issue.

Reply:
Looks like that is how the generated `.pb.go` files refer to ittoo; I daresay that `goimports` is getting fooled by that.